pkg/asset/cluster/tfvars.go

```diff
@@ -686,6 +690,54 @@ func (t *TerraformVariables) Generate(parents asset.Parents) error {
 			Filename: fmt.Sprintf(TfPlatformVarsFileName, platform),
 			Data:     data,
 		})
+	case alibabacloud.Name:
+		client, err := icalibabacloud.NewClient(installConfig.Config.Platform.AlibabaCloud.Region)
+		if err != nil {
+			return err
+		}
+		bucket := fmt.Sprintf("%s-bootstrap", clusterID.InfraID)
+		auth := alibabacloudtfvars.Auth{
+			AccessKey: client.AccessKeyID,
+			SecretKey: client.AccessKeySecret,
+		}
+
+		// TODO AlibabaCloud: Master and worker machine info
+		masters, err := mastersAsset.Machines()
+		if err != nil {
+			return err
+		}
+		masterConfigs := make([]*alibabacloudprovider.MachineProviderSpec, len(masters))
+		for i, m := range masters {
+			masterConfigs[i] = m.Spec.ProviderSpec.Value.Object.(*alibabacloudprovider.MachineProviderSpec)
+		}
+		workers, err := workersAsset.MachineSets()
+		if err != nil {
+			return err
+		}
+		workerConfigs := make([]*alibabacloudprovider.MachineProviderSpec, len(workers))
+		for i, w := range workers {
+			workerConfigs[i] = w.Spec.Template.Spec.ProviderSpec.Value.Object.(*alibabacloudprovider.MachineProviderSpec)
+		}
+
+		data, err := alibabacloudtfvars.TFVars(
+			alibabacloudtfvars.TFVarsSources{
+				// TODO: AlibabaCloud: later PR
```

Comment:
The `ignition_stub` for the bootstrap and the masters needs to be populated here.

Reply:
We will change it tomorrow